src/naev.c

```diff
@@ -489,6 +489,29 @@ int main( int argc, char** argv )
    /* flushes the event loop since I noticed that when the joystick is loaded it
     * creates button events that results in the player starting out acceling */
    while (SDL_PollEvent(&event));
+
+   /* Incomplete game note. */
+   if (!conf.firstran) {
```

Comment:
It might be worth making it be some sort of value like the major release version so we can display the message every major release.

Reply:
Sure, I can do that.

🕵️

Reply:
We never change the major release. 😉 So I used the short version number string instead.

🕷️